src/storage-controller/src/lib.rs

```diff
@@ -156,119 +152,15 @@ mod statistics;
 pub static METADATA_COLLECTION: TypedCollection<proto::GlobalId, proto::DurableCollectionMetadata> =
     TypedCollection::new("storage-collection-metadata");
 
-pub static METADATA_EXPORT: TypedCollection<proto::GlobalId, proto::DurableExportMetadata> =
```

Comment:
Do we care about adding the functionality into the stash to delete collections, so we can clean up the data associated with this?

Reply:
I thought about that. How would that look like? Are stash collections represented as separate tables? I haven't been following closely how stash stores things

Reply:
No there all stored in a single table and their mapping is stored in a separate table. If you were interested we would probably add the functionality to the stash and then you can call it on startup. Or we can add it into the stash debug tool.